### ISSUE TITLE
Documentation index should respect versions

### ISSUE THREAD

**Issue by camfindlay** [silverstripe/silverstripe-docsviewer#59]
Currently this displays everything for every version of documentation (if using the multiversions).

This ideally would return the index for the currently selected version of the documentation and display the select version interface.

**Comment by SpiritLevel** [comment]
Ideally ? More like crucially !! :P

**Comment by camfindlay** [comment]
Is this what your #77 fixes?

**Comment by SpiritLevel** [comment]
No...#77 only causes the HOME menu item in the side bar to return to the home page of the currently selected version instead of its old behaviour of always going back to the stable version (3.2).  

I haven't yet thought about #59. I was just emphatically agreeing with you! The call to $Link(all) at the bottom of DocumentationSidebar.ss is the problem behind #59.
